app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] As the ERD diagram shows,  you must have for has_many association  a`foreign_key: 'author_id'`. Kindly implement them for your, :posts, :comments, and :likes as well :).

Reply:
Thank you, I deleted those ones when I did the migration so I could add them again later because of some error that I was having, but I forgot to add them again 😅